packages/trader/src/sass/app/modules/smart-chart.scss

```diff
@@ -5,7 +5,7 @@
 
     &__spot {
         position: absolute;
-        bottom: -11px;
+        top: -11px;
```

Comment:
Why is this a negative value? can we give it `bottom: 1.1 rem`?

Reply:
Bottom is not the opposite of top in terms of positioning in css. If I use bottom 1.1rem, it will move it relatively to the bottom of the outer block, while I need to move it relatively to top of outer block.